.github/workflows/e2e.yml

```
@@ -84,6 +105,17 @@ jobs:
           java-version: "17"
 
+      - name: Setup Android SDK
+        uses: android-actions/setup-android@v3
+
+      - name: Install NDK 28.2.x
+        run: |
+          # Project requires NDK 28.2.x for CMake LTO fixes
+          sdkmanager --install "ndk;28.2.13676358"
+
+      - name: Create output directory
+        run: mkdir -p $HOME/output
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:1:10: Double quote to prevent globbing and word splitting

(shellcheck)